test/azure-blob-storage.ts

```diff
@@ -1,10 +1,7 @@
 import assert from 'node:assert'
 import crypto from 'node:crypto'
-import fs from 'node:fs'
-import { tmpdir } from 'node:os'
-import { join } from 'node:path'
-import { mock, test } from 'node:test'
-import fsbs from 'fs-blob-store'
```

Comment:
Testing with reading/writing the real file on the filesystem doesn't look reliable. One test pass depends on the other test run, tests fail if you change the order and using another lib in the azure storage mock adds unnecessary complexity. It's enough to check if the mock was called with the expected arguments. 
I suggest to refactor the other test suites the same way.

Reply:
This is a good point.